.travis.yml

```diff
@@ -0,0 +1,44 @@
+language: android
+jdk: oraclejdk8
+env:
+  global:
+    - ANDROID_API_LEVEL=25
+    - ANDROID_BUILD_TOOLS_VERSION=25.0.0
+android:
+  licenses:
+    - 'android-sdk-preview-license-.+'
+    - 'android-sdk-license-.+'
+    - 'google-gdk-license-.+'
+  components:
+    - tools
+    - tools
+    - platform-tools
+    - extra-google-google_play_services
+    - extra-google-m2repository
+    - extra-android-m2repository
+    - build-tools-25.0.0
+    - android-25
+    - android-22
+    - sys-img-armeabi-v7a-android-22
+
+
+before_install:
+  - yes | sdkmanager "platforms;android-27"
+  - yes | sdkmanager "platforms;android-28"
+  - pip install --user codecov
+
+before_script:
+  - echo no | android create avd --force -n test -t android-22 --abi armeabi-v7a
+  - emulator -avd test -no-audio -no-window &
+  - android-wait-for-emulator
+  - adb shell input keyevent 82 &
+
+
+script:
+  - ./gradlew build check
+  - if [ -e ./gradlew ]; then ./gradlew jacocoTestReport;else gradlew jacocoTestReport;fi
```

Comment:
Please confirm that this is not a syntax error for 
` else ./gradlew jacocoTestReport;fi `
note the ./gradlew /= gradlew